src/main/resources/application.yml

```diff
@@ -76,11 +76,13 @@ notify:
     referral-sent: "336f2158-d130-4c63-a59b-da12df15a5a3"
     referral-assigned: "f0fe2fce-df0f-46c0-839a-822037d39c28"
     action-plan-submitted: "83743b43-9abb-4ffb-913d-dc583ab22dbf"
+    appointment-not-attended: "21b14c86-30d2-4bf2-b355-8f3deb62ef69"
 
 interventions-ui:
   locations:
     sent-referral: "/service-provider/referrals/{id}"
     submit-action-plan: "/action-plan/{id}/toBeSpecified"
+    appointment-not-attended: "/appointment/{id}/toBeSpecified"
```

Comment:
not a blocker - is it better to have these links point to the pp dashboard, so at least they actually work, rather than to a 404 page?

Reply:
done 

https://dsdmoj.atlassian.net/browse/IC-1482
https://dsdmoj.atlassian.net/browse/IC-1483